torch/optim/swa_utils.py

```diff
@@ -56,16 +109,15 @@ class AveragedModel(Module):
         >>> # Update bn statistics for the swa_model at the end
         >>> torch.optim.swa_utils.update_bn(loader, swa_model)
 
-    You can also use custom averaging functions with `avg_fn` parameter.
+    You can also use custom averaging functions with the `avg_fn` or `multi_avg_fn` parameters.
     If no averaging function is provided, the default is to compute
-    equally-weighted average of the weights.
+    equally-weighted average of the weights (SWA).
 
     Example:
         >>> # xdoctest: +SKIP("undefined variables")
         >>> # Compute exponential moving averages of the weights and buffers
-        >>> ema_avg = lambda averaged_model_parameter, model_parameter, num_averaged: (
-        ...                 0.1 * averaged_model_parameter + 0.9 * model_parameter)
-        >>> swa_model = torch.optim.swa_utils.AveragedModel(model, avg_fn=ema_avg, use_buffers=True)
+        >>> swa_model = torch.optim.swa_utils.AveragedModel(model,
+        >>>             torch.optim.swa_utils.get_ema_multi_avg_fn(0.9), use_buffers=True)
```

Comment:
Would be good to update the notes below to include multi_avg_fn + EMA

Reply:
Done